### PR TITLE
Updated / added german translation.

### PR DESCRIPTION
**Pull request by ThomasVie**
Here are some commits which are translation related (mostly).
Additionally, I saw some more warnings. How to fix them:

lxqt-config/lxqt-config-brightness/main.cpp:32: tr() cannot be called without context
lxqt-config/lxqt-config-brightness/main.cpp:35: tr() cannot be called without context
lxqt-config/lxqt-config-brightness/main.cpp:38: tr() cannot be called without context

### CODE REVIEW

**Comment by ThomasVie** [comment]
Rebased. I also fixed the warnings "tr() cannot be called without context".

**Comment by palinek** [comment]
gtm

**Comment by pmattern** [comment]
IMO we've got a bit too many commits here, in fact their number exceeds the one of changed file.
I think it would be better to squash at least some of them, e. g. summarizing actions like add context of lupdate, add new files or so.

**Comment by ThomasVie** [comment]
I could squash it next weekend.

File: lxqt-config-monitor/main.cpp [hunk 1]
```diff
@@ -52,7 +52,7 @@ int main(int argc, char** argv)
     // Command line options
     QCommandLineParser parser;
     QCommandLineOption loadOption(QStringList() << "l" << "loadlast",
-            app.tr("Load last settings."));
+            LXQt::SingleApplication::tr("Load last settings."));
```
Comment (paulolieuthier) [inline lxqt-config-monitor/main.cpp]: Are you sure this is necessary?

Comment (ThomasVie) [inline lxqt-config-monitor/main.cpp]: Yes, without it I get the following (lupdate version 5.5.1):
/home/src/lxqt-config/lxqt-config-monitor/main.cpp:55: tr() cannot be called without context

**Comment by ThomasVie** [comment]
Rebased, squashed commits.

**Comment by agaida** [comment]
Must be modified to match our repo structure - sorry, overseen it

**Comment by agaida** [comment]
@ThomasVie - please rebase/rework

**Comment by ThomasVie** [comment]
I don't have a suitable development environment anymore, so I am unable to check a possible rebased result.

**Pull request closed without merging**